src/main/resources/kairosdb.conf

```diff
@@ -59,21 +59,42 @@ kairosdb: {
-		# the auth_module_name below should match the name outside the curly braces in the respective conf file.
-		# The basics of how to configure two or more modules to work together in logic and/or combinations can be found here https://docs.oracle.com/javase/8/docs/api/javax/security/auth/login/Configuration.html and here https://github.com/rundeck/rundeck/wiki/Multiple-authentication-modules
-		#auth_module_name: "basicAuth"
-
 		# To enable thread pooling uncomment the following lines and specify the limits
 		#threads.queue_size: 6000
 		#threads.min: 1000
 		#threads.max: 2500
 		#threads.keep_alive_ms: 10000
 	}
 
+	# To enable Jetty auth use the settings below. The two required for auth is the module_name Service.jetty_auth. The others are set to the below defaults in PulseBasicAuthenticatorModule
+	# Service.jetty_auth as well as module_name in jetty_auth section must be configured and uncommented for auth to be enabled. Also read further details above module_name for configuration of auth
+	#service.jetty_auth: "org.kairosdb.auth.PulseBasicAuthenticatorModule"
+    #jetty_auth: {
+
+    # KairosDB now uses the Jetty JAAS Framework to do Authentication. You can extend the Jetty JAAS framework to do many kinds of authentication.
+    # Under the auth folder you will find 1 file that allows configuration of LDAP auth
+    # Samples of how to configure the various auth versions can be found here https://www.eclipse.org/jetty/documentation/current/jaas-support.html
+    # ldap-loginModule.conf will configure the ldap module.
+    # the auth_module_name below should match the name outside the curly braces in the respective conf file.
+    # The basics of how to configure two or more modules to work together in logic and/or combinations can be found here https://docs.oracle.com/javase/8/docs/api/javax/security/auth/login/Configuration.html and here https://github.com/rundeck/rundeck/wiki/Multiple-authentication-modules
+    # Service.jetty_auth as well as module_name in jetty_auth section must be configured and uncommented for auth to be enabled.
+    #module_name: "cachingLDAPAuth"
+
+    # To enable softAuthentication(allows no credentials, so services can enable auth over time)LDAPAuth functionality set the following property to true.
+    # Specifically enables the org.kairosdb.auth.PulseDefaultAuthenticatiorFactory and org.kairosdb.auth.PulseBasicAuthenticator to be used in conjunction with the org.kairosdb.auth.PulseAuthModulee.
+    # also requires setting the appropriate configuration for the LDAPLoginModule in the ldap-loginModule.conf file above including the file property for the PropertyFile LoginModule.
+    #use_soft_authentication: false
+
+    # The amount of time, in minutes, a user will remain in the cache after logging in
+    #user_cache_duration: 60
+
+    # The amount of time, in minutes, a user + password combination will be locked out of the system from failed logins
```

Comment:
From a progression perspective these two `lockout` variables should be swapped (so as
you read through the example you see `lockout_cache_threshold` first, which
tells you how many failed attempts get you locked, _then_
`lockout_cache_duration` tells you how long to wait before _unlocking_ that
user.)

Reply:
Fixed on latest commit